test-packages/e2e-tests/test/openapi.spec.ts

```diff
@@ -32,6 +32,13 @@ describe('openapi request builder', () => {
     const postCreateCount = await countEntities();
     expect(postCreateCount).toEqual(preCreateCount + 1);
   });
+
+  it('executes request with quote in property', async () => {
+    const response = await EntityApi.getAllEntities({
+      enumStringParameter: 'value1'
```

Comment:
```suggestion
      enumStringParameter: "valueWith'Quote"
```

Reply:
ups, this was a test I commited by accident.